src/network/serverpackethandler.cpp

```diff
@@ -1203,10 +1205,11 @@ void Server::handleCommand_Interact(NetworkPacket* pkt)
 				infostream << "Server: Not finishing digging: Node not found."
 						<< " Adding block to emerge queue."
 						<< std::endl;
-				m_emerge->enqueueBlockEmerge(pkt->getPeerId(),
+				m_emerge->enqueueBlockEmerge(peer_id,
 					getNodeBlockPos(p_above), false);
 			}
 
+			ItemStack playeritem = playersao->getWieldedItemOrHand();
```

Comment:
why ? it's seems unused in next code

Reply:
Thanks, I forgot to undo this temporary change.